processor/redactionprocessor/config.go

```diff
@@ -3,6 +3,17 @@
 
 package redactionprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/redactionprocessor"
 
+import "fmt"
+
+type HashFunction string
+
+const (
+	NONE HashFunction = ""
```

Comment:
Not sure if 'NONE' is all caps intentionally, if not I would just put it with the usual Go style
```suggestion
	None HashFunction = ""
```